src/docs/patterns/appbar.mdx

```diff
@@ -35,3 +37,37 @@ In some situations, you may want to present more information in an app bar, such
 <DemoCard repository={'search-bar'} angular react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/app-bar/search-bar/index.tsx&initialpath=search'} ionic reactNative />
 
 A very common pattern seen in many types of applications is the ability to search for data on a page. This is achieved by placing a search icon/button in the app bar. When this button is clicked, a white Search Bar slides in from the right to cover the default toolbar. As the user enters text into the search field, the data is filtered in real-time.
+
+<Divider />
+
+# Design Specifications
+
+For additional instructions, refer to [Material Design's design specs on "App bars: top"](https://material.io/components/app-bars-top#specs).
+
+## Mobile
+
+<ImageGrid images={[Specs1Image]} regularWidth />
```

Comment:
Do we need to mention color here? Inherit is the default CSS behavior and does it make sense to a designer?

Should we instead say something like "white[50] on dark backgrounds and black[500] on light backgrounds"?

Reply:
I should probably just remove it unless there are certain colors not immediately obvious.

Reply:
Adding to that, I think the design spec is not only for designers... I am trying to find the best language for both designers and developers.

Reply:
I would probably leave this out like you said unless it's a non-obvious color.